packages/wallet-sdk/src/sign/scw/utils/getAccountIndex.ts

```diff
@@ -0,0 +1,37 @@
+import { Client, Hex } from 'viem';
+import { readContract } from 'viem/actions';
+
+import { abi } from './constants.js';
+import { standardErrors } from ':core/error/errors.js';
+
+export async function getAccountIndex({
```

Comment:
nit: getOwnerIndex or getAccountOwnerIndex work better here

Reply:
+1 was debating renaming the properties passed to createSmartAccount. Will do that here as well